File: Maths/Algebra/LCM/LCM.java
// Java program to find LCM of two numbers.
class Test
{
	
	static int gcd(int a, int b)
	{
		if (a == 0)
			return b; 
		return gcd(b % a, a); 
	}
	
	
	static int lcm(int a, int b)
	{
		return (a / gcd(a, b)) * b;
	}
	
	
	public static void main(String[] args) 
	{
		int a = 15, b = 20;
		System.out.println("LCM of " + a +
						" and " + b + 
					" is " + lcm(a, b));
	}
}

File: Maths/Algebra/LCM/LCM.js
let lcm = (n1, n2) => {
  //Find the smallest and biggest number from both the numbers
  let lar = Math.max(n1, n2);
  let small = Math.min(n1, n2);

  //Loop till you find a number by adding the largest number which is divisble by the smallest number
  let i = lar;
  while (i % small !== 0) {
    i += lar;
  }

  //return the number
  return i;
};

console.log(lcm(12, 13));
console.log(lcm(9, 3));
